Sources/SwiftFortuneWheel/Utils/Audio/AudioPlayerManager.swift

```diff
@@ -144,7 +145,9 @@ class AudioPlayerManager {
     private func createNewPlayerAttachedToEngine() -> AudioPlayer {
         let player = AudioPlayer()
         engine.attach(player.node)
-        engine.connect(player.node, to: engine.mainMixerNode, format: nil)
+        // according to this: https://stackoverflow.com/a/55505717/3687801, we will defer connecting nodes to engine up until play command of player node
```

Comment:
not a good idea to remove the connection to engine from this method. It can cause sound delay.